.eleventy.js

```diff
@@ -1,16 +1,30 @@
 module.exports = function (eleventyConfig) {
-  // Plugins
+  // Options to customise the appearance of your design history
+  // https://x-govuk.github.io/govuk-eleventy-plugin/options/
   eleventyConfig.addPlugin(require('govuk-eleventy-plugin'), {
+    brandColour: '#28a',
+    fontFamily: 'system-ui, sans-serif',
     stylesheets: [
       '/styles/application.css'
     ],
     headingPermalinks: true,
+    homeKey: 'X-GOVUK',
     header: {
+      organisationLogo: 'x-govuk',
+      organisationName: 'X-GOVUK',
```

Comment:
I think I'd prefer this to have no logo, as it would be a simpler template to clone begin using?
What do you think?

Maybe like this?
<img width="1006" alt="Screenshot 2022-10-14 at 17 35 37" src="https://user-images.githubusercontent.com/319055/195896844-96ca7052-0201-49bb-bf71-2fd1bd97eb49.png">